lang/en/tool_heartbeat.php

```diff
@@ -25,13 +25,15 @@
 $string['pluginname'] = 'Heartbeat';
 $string['errorlog'] = 'Error log period';
 $string['errorlogdesc'] = 'To help ensure that all web server logging is working we can emit an intermittent message to the error_log. Set this to 0 to turn it off.';
-$string['testing'] = 'Test heartbeat';
-$string['testingdesc'] = 'You can use this to temporarily fake a warn or error condition to test that your monitoring is correctly working end to end.';
 $string['normal'] = 'Normal monitoring';
 $string['progress'] = 'Progress bar test';
 $string['progresshelp'] = 'This tests that all the various output buffers in the entire stack are corrent including but not limited to php, ob, gzip/deflat, varnish, nginx etc';
-$string['testwarning'] = 'Fake a warning';
+$string['testabort'] = 'Test abort';
+$string['testaborthelp'] = '<p>This tests wether the whole stack is correctly passing a request cancel signal from the browser to php through whatever layers are in the middle, such as load balancers, cdn\'s, caches, php-fm etc.</p>';
```

Comment:
Might be worth explaining what the stages mean, what to expect (what is bad vs what is good)